scripts/check-signed-off-by.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+MAIN_BRANCH=master
+
+# ignores last commit because travis/gh creates a merge commit
+commits=$(git log --format=%h ${MAIN_BRANCH}..HEAD | tail -n +2)
```

Comment:
How about
```
commits=$(git log --format=%h ${MAIN_BRANCH}..HEAD~1)
```
since git can do the parent thing and stuff.

Reply:
Done!

Reply:
And undone!

Reply:
It fails with `~1` and it was failing before for me with `^`. I suppose it has todo with the latest PR commit that GH creates which might not have the right parent<-> association. Not sure.

Reply:
Have you found an example where this is failing? I would argue that tail +2 is doing the wrong thing, and HEAD~1 is doing the right thing. I'm curious what is going on with that.

Reply:
https://travis-ci.org/runtimeco/mcuboot/builds/272715788

Reply:
My suspicion here is that Travis only makes a merge if it has to. If the tree can be fast forwarded, it will be. Your "tail" version is then ignoring the latest commit in the series, and my suggestion ignores all of them. Neither is correct.

I think we need to figure out either how to make travis use merge --no-ff, or be much smarter in what we check for.

```
git log --pretty=raw ${MAIN_BRANCH}..
exit 1
```
in a travis build might be revealing, do it where it needs to be rebased, and where it doesn't.